src/analytics/lambdas/custom-resource/create-schemas.ts

```diff
@@ -394,7 +394,7 @@ const createDatabaseBIUser = async (redshiftClient: RedshiftDataClient, credenti
   props: CreateDatabaseAndSchemas) => {
   try {
     await executeStatementsWithWait(redshiftClient, [
-      `CREATE USER ${credential.username} PASSWORD 'sha256|${credential.password}'`,
+      `CREATE USER ${credential.username} PASSWORD '${credential.password}'`,
```

Comment:
if you're using sha256 algorithm, why not encoding the password with that hash algorithm?